projects/storefrontapp-e2e-cypress/cypress/e2e/regression/user_access/navigation-login-a11y.e2e.cy.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as login from '../../../helpers/login';
+import { viewportContext } from '../../../helpers/viewport-context';
+
+function assertNavigationButtonsAttributes(buttonsSelector: string) {
+  cy.get(buttonsSelector).each(($btn) => {
+    const btnAriaLabel = $btn.attr('aria-label');
+    cy.wrap($btn)
+      .should('have.attr', 'title', `${btnAriaLabel} Menu`)
+      .should('have.attr', 'aria-label', btnAriaLabel)
+      .should('have.attr', 'aria-controls', btnAriaLabel);
+  });
+}
```

Comment:
Not sure I understand this part: We are grabbing the value of the button's aria label and then checking if it has that value?